### ISSUE TITLE
[Shows] Add/Edit Show Phone Number

### ISSUE THREAD

**Issue by tonyzeoli** [netmix/radio-station#212]
**Is your feature request related to a problem? Please describe.**

A user asked if we could create a field or Station Phone Number for call-in requests.

**Describe the solution you'd like**

We should give the option to Add/Edit Show Phone Number. Optional display on Show Page and Current Show On Air Widget.

Display: Call In: +1 (000) 000-0000 (or U.S. numbers) and allow or international numbers.

Use "tel:" in the link to allow the user to click the link from mobile and call.

**Describe alternatives you've considered**

I've thought about also adding a link to chat messenger, but that's a PRO feature. We should add Chat w/ the DJ to PRO.

**Additional context**

**Comment by tonyzeoli** [comment]
@majick777 Moving this to 2.3.3.

**Comment by larryl79** [comment]
Or even viber / whatsapp number field as well. ( My friends radio using just viber for message into the show )

**Comment by majick777** [comment]
@tonyzeoli I've added the Phone Number field to Shows. Since validating possible international numbers combinations could prove a nightmare, I've put in character validation instead: `0-9`, `.`, `-`, '#', `(`, `)`, ` `

I've also added a Station phone number field in the plugin Settings, and a checkbox to enable the use of this where a Show does not have a phone number specified.

The phone number displays as a click to call icon on the Show page as well as spelling out the actual number visibly in the Show info section below. 

I just saw your note on adding the option to display this in the Current Show Widget, so I will leave this open until I add that.
